clients/bmp_client.go

```diff
@@ -79,3 +79,181 @@ func (bc *bmpClient) SlPackages() (SlPackagesResponse, error) {
 
 	return response, nil
 }
+
+func (bc *bmpClient) Stemcells() (StemcellsResponse, error) {
+	path := fmt.Sprintf("%s/%s", bc.url, "/stemcells")
+	responseBytes, errorCode, err := bc.httpClient.DoRawHttpRequest(path, "GET", &bytes.Buffer{})
+	if err != nil {
+		errorMessage := fmt.Sprintf("bmp: could not calls /stemcells on BMP server, error message '%s'", err.Error())
+		return StemcellsResponse{}, errors.New(errorMessage)
+	}
+
+	if slcommon.IsHttpErrorCode(errorCode) {
+		errorMessage := fmt.Sprintf("bmp: could not call /stemcells on BMP server, HTTP error code: '%d'", errorCode)
+		return StemcellsResponse{}, errors.New(errorMessage)
+	}
+
+	response := StemcellsResponse{}
+
```

Comment:
no need for new line here. It's setup for the next statements... In general group related statements together.